3-platform/order-ms/app/src/main/java/com/core/platform/orderms/app/OrderUseCase.java

```diff
@@ -0,0 +1,21 @@
+package com.core.platform.orderms.app;
+
+import com.core.platform.orderms.domain.model.OrderRecord;
+import com.core.platform.orderms.domain.persistent.OrderRepository;
+import com.core.platform.orderms.domain.service.OrderService;
+
+public final class OrderUseCase {
```

Comment:
we don't need these dummy classes, leave packages empty, or add package-info.java to reflect package structure in git.